packages/http/src/validator/deserializers/__tests__/registry.spec.ts

```diff
@@ -1,11 +1,13 @@
-import { HttpParamDeserializerRegistry } from '..//HttpParamDeserializerRegistry';
-import { DeserializeHttpQuery, IHttpParamStyleDeserializer } from '../types';
+import { HttpParamStyles } from '@stoplight/types/http.d';
```

Comment:
@pytlesk4 why are you importing `http.d`? That won't work imo. Just remove `.d` :)

Reply:
That was a workaround to get it to work for me last night. I don't know why it did, but I could have been dreaming haha. Anyways, I have undone that. About to push my changes.